packages/examples/fortune-v2/reputation-oracle/server/src/modules/reputation/reputation.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Entity } from 'typeorm';
+
+import { NS } from '../../common/constants';
+import { BaseEntity } from '../../database/base.entity';
+import { ReputationEntityType } from '../../common/decorators';
+
+@Entity({ schema: NS, name: 'reputation' })
+export class ReputationEntity extends BaseEntity {
+  @Column({ type: 'int' })
+  public chainId: number;
```

Comment:
Shouldn't it use the enum?
```suggestion
  public chainId: ChainId;
```